packages/editor/src/client/locales/en.json

```diff
@@ -785,7 +786,7 @@
         "token": "Token",
         "unknown": "Unknown",
         "unnamed": "Unnamed",
-        "URL": "URL",
+        "URL": "URL: https://www.url.com",
```

Comment:
Necessary to put the test data in translations? Maybe should be a separate test category rather than storing it under the regular panels?

Reply:
It's no test data it's a placeholder to show how the format should be like